examples/v1beta1/taskruns/alpha/emit-array-results.yaml

```diff
@@ -0,0 +1,30 @@
+kind: Task
+apiVersion: tekton.dev/v1beta1
+metadata:
+  name: write-array
+  annotations:
+    description: |
+      A simple task that writes array
+spec:
+  results:
+    - name: array-results
+      type: array
+      description: The array results
+  steps:
+    - name: write-array
+      image: bash:latest
+      script: |
+        #!/usr/bin/env bash
+        echo -n "[\"hello\",\"world\"]" | tee $(results.array-results.path)
+    - name: print-array
+      image: bash:latest
+      script: cat $(results.array-results.path)
```

Comment:
this could be a little more robust example by introducing validation that `$(results.array-results.path)` matches the expected value ...

Reply:
Sure!

Reply:
I added some check, is it ok?